core/client/tests/acceptance/settings/tags-test.js

```diff
@@ -46,9 +48,11 @@ describe('Acceptance: Settings - Tags', function () {
 
     beforeEach(function () {
         application = startApp();
+        errorOverride();
```

Comment:
I think in this case it would be better to add the override only to the tests that we know will throw an error - we don't want to hide errors in tests where we aren't expecting them.